src/status_im/contexts/wallet/edit_account/view.cljs

```diff
@@ -114,3 +113,5 @@
                                                                   :new-value   chain-ids}))
                                                  :watch-only? watch-only?}])}]))
            :container-style style/data-item}]]))))
+
+(defn view [] [:f> f-view])
```

Comment:
No longer needed :)

Reply:
there's a hook inside though
```clj
{:keys [keyboard-shown]} (hooks/use-keyboard) ;; line 66
```

Reply:
Funcitonal components have been enabled by default in a recent PR, so we no longer need to differenciate them

Reply:
ahh right, thanks :+1: